### DISCUSSION TITLE
Error in installing homebrew for first time

### DISCUSSION THREAD

**Discussion by shahdivyam** [Homebrew/discussions#5055] [Getting started]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Trying to install homebrew with following command ` /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"` 

getting below error `Failed to download https://formulae.brew.sh/api/cask.jws.json!
Failed to download https://formulae.brew.sh/api/formula_tap_migrations.jws.json!
Failed to download https://formulae.brew.sh/api/cask_tap_migrations.jws.json!
curl: (6) Could not resolve host: formulae.brew.sh
Error: Failure while executing; `/usr/bin/env /opt/homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --user-agent Homebrew/4.2.3\ \(Macintosh\;\ arm64\ Mac\ OS\ X\ 13.5\)\ curl/8.1.2 --header Accept-Language:\ en --fail --progress-bar --silent --remote-time --output /Users/divyam/Library/Caches/Homebrew/api/cask.jws.json --location --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.2.3\ \(Macintosh\;\ arm64\ Mac\ OS\ X\ 13.5\)\ curl/8.1.2 --header Accept-Language:\ en --fail --progress-bar --silent --compressed --speed-limit 100 --speed-time 5 https://formulae.brew.sh/api/cask.jws.json` exited with 6. Here's the output:
curl: (6) Could not resolve host: formulae.brew.sh

Failed during: /opt/homebrew/bin/brew update --force --quiet` 

Any inputs on fixing this would be appreciated!

**Comment by SMillerDev** [comment]
It seems your network is blocked from contacting formulae.brew.sh. Unfortunately there is nothing homebrew can do to fix that.
